Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: test
+
+test:
+	redis-cli flushall && FAKEREDIS=false npm test
```

Comment:
please don't commit this.  It's specific to how you are testing, not the test suite as defined